### ISSUE TITLE
Are the C files necessary?

### ISSUE THREAD

**Issue by sondrele** [RustyGecko/emdrv#4]
There's a few C-files in the source code, are they located in efm32-common as well?

Are these files necessary in the src folder?:
- https://github.com/RustyGecko/emdrv/blob/master/src/i2c1drv.c
- https://github.com/RustyGecko/emdrv/blob/master/src/i2c1drv.h
- https://github.com/RustyGecko/emdrv/blob/master/src/i2c1drvconfig.h
- https://github.com/RustyGecko/emdrv/blob/master/src/i2cdrvconfig.h

**Comment by havardh** [comment]
Yes, they are referenced from here: https://github.com/RustyGecko/emdrv/blob/master/src/i2c.rs#L13.

Ideally these should be moved into the SensorTracker application, and the code here should reference `i2cdrv.c` and friends from `emdrv`

**Comment by sondrele** [comment]
Possible future task then... :)
